src/subcategories/unspecified.ts

```diff
@@ -0,0 +1,12 @@
+import { makeEnum } from '@transcend-io/type-utils';
+
+export const UnspecifiedSubCategory = makeEnum({
+  /** Fallback subcategory */
+  Unspecified: 'unspecified',
```

Comment:
wait i thought we were going to name them "Other"

Reply:
We will but that's a frontend thing, we need distinct names for the enum values